### ISSUE TITLE
Had the same Issue as Alexy1113 but resolved it, Which brought about the same issue that colapsys had

### ISSUE THREAD

**Issue by Nicholas55555** [Sh3llcod3/Airscript-ng#7]
What I did was uninstall all of the dependencies (option 14) and reinstall them with github (option 7)

Now I have this bug: /bin/sh: 1: /root/.airscriptNG/air/aircrack-ng/src/airodump-ng: Permission denied

but hey, at least its installed. Perhaps you can resolve Alexy1113's issue and this one and prioritise colapsys's issue.

**Comment by Nicholas55555** [comment]
This is on arm64

**Comment by Sh3llcod3** [comment]
Right, I'll take a look asap. If you look at the dev branch, the classes to install dependencies are fixed already, so when the rewrite is finished this will not be an issue anymore hopefully.
